display_list/BUILD.gn

```diff
@@ -123,6 +126,7 @@ if (enable_unittests) {
       ":display_list_fixtures",
       "//flutter/display_list/testing:display_list_testing",
       "//flutter/testing",
+      "//third_party/skia",
```

Comment:
Why is this needed? I'm guessing if the spy stuff gets moved out of here into the embedder source directories then it doesn't need to be added here, but I would have thought that you could implement an OpReceiver without having to add this dependency...?

Reply:
This is for the unittest, to generate a concrete DlImage for testing.